src/SOS/SOS.Hosting/LLDBServices.cs

```diff
@@ -79,6 +80,8 @@ public LLDBServices(SOSHost soshost)
             builder = AddInterface(IID_ILLDBServices2, validate: false);
             builder.AddMethod(new LoadNativeSymbolsDelegate2(LoadNativeSymbols2));
             builder.AddMethod(new AddModuleSymbolDelegate(AddModuleSymbol));
+            builder.AddMethod(new GetModuleInfoDelegate(GetModuleInfo));
```

Comment:
For the older ILLDServices interface I'm trying to stay compatible so the sosplugin from diagnostics repo will work with older libsos.so's. 

But until we actually release a stable version the new ILLDBServices2 interface can change and maybe even after that because going forward, the plugin and sos are shipped together and more tightly bounded.